tests/php/test-amp-facebook-embed.php

```diff
@@ -184,13 +184,6 @@ public function get_raw_embed_dataset() {
 				'<amp-facebook-comments layout="responsive" width="600" height="400" data-href="https://developers.facebook.com/docs/plugins/comments#configurator" data-numposts="5"></amp-facebook-comments>',
 			],
 
-			'comments_full_width'   => [
-				'
-					<div class="fb-comments" data-href="https://developers.facebook.com/docs/plugins/comments#configurator" data-width="100%" data-numposts="5"></div>
-				',
-				'<amp-facebook-comments layout="fixed-height" width="auto" height="400" data-href="https://developers.facebook.com/docs/plugins/comments#configurator" data-numposts="5"></amp-facebook-comments>',
-			],
-
```

Comment:
This should not have been removed. See https://github.com/ampproject/amp-wp/pull/2776